manipulation/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java-gradle-plugin' // applies "java" plugin and adds gradleApi() to dependencies
+}
+
+group 'org.jboss.pme.manipulation'
```

Comment:
How about
```suggestion
group 'org.jboss.gradle.plugin.build-manipulator`
```

Reply:
Settled on `org.jboss.gm.manipulation` for now. There will be opportunity to rename things better once we have something that works :)